python-numpy-basics/Classes_and_Objects.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>Add rich text , instead of image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
i will require images to explain the allocation of memory for an object and its instance namespace anyways

Reply:
<p>This image has no visual illustration , and can be written in markdown . So make it rich text.</p><p> </p>